src/conversion-settings.ts

```diff
@@ -180,12 +180,15 @@ export function createDefaultConversionSettings(
   const importedFiles =
       IterableX
           .from(analysis.getFeatures({kind: 'import', externalPackages: false}))
-          .map((imp) => imp.url)
+          .filter((imp) => !imp.kinds.has('html-script-back-reference'))
+          .map(
+              (imp) =>
+                  analyzer.urlResolver.relative(imp.url) as string | undefined)
           .filter(
-              (url) =>
-                  !(url.startsWith('bower_components') ||
-                    url.startsWith('node_modules')));
-  const includes = new Set(importedFiles);
+              (url) => !!url &&
```

Comment:
I prefer explicit checks so we know what values to expect here. `url !== undefined`?